test/suites/integration/expiration_test.go

```diff
@@ -109,15 +101,8 @@ var _ = Describe("Expiration", func() {
 		env.Monitor.Reset() // Reset the monitor so that we can expect a single node to be spun up after expiration
 
 		// Eventually the node will be tainted, which means its actively being disrupted
-		Eventually(func(g Gomega) {
-			g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(node), node)).Should(Succeed())
-			_, ok := lo.Find(node.Spec.Taints, func(t corev1.Taint) bool {
-				return t.MatchTaint(&karpv1.DisruptedNoScheduleTaint)
-			})
-			g.Expect(ok).To(BeTrue())
-		}).Should(Succeed())
-
-		env.EventuallyExpectCreatedNodeCount("==", 2)
+		env.EventuallyExpectTaintedNodeCount("==", 1)
```

Comment:
Feel this could be more descriptive on `EventuallyExpectTaintedForDisruptionNodeCount`, or something like that to get across that this is really checking about `disruption` being triggered on the nodes, and tainted accordingly. 

However, not for this PR

Reply:
Resolving as not for this PR.